core/templates/components/oppia-angular-root.component.ts

```diff
@@ -374,6 +374,7 @@ import { StateCardObjectFactory } from
 import { StateClassifierMappingService } from
   'pages/exploration-player-page/services/state-classifier-mapping.service';
 import { StateContentService } from 'components/state-editor/state-editor-properties-services/state-content.service';
+import {StateLinkedSkillIdService} from 'components/state-editor/state-editor-properties-services/state-skill.service';
```

Comment:
Changes to this file are not required anymore.

Reply:
Done.